tests/unit/test_solvers/test_idaklu_jax.py

```diff
@@ -0,0 +1,901 @@
+#
+# Tests for the KLU-Jax interface class
+#
+from tests import TestCase
+from parameterized import parameterized
+
+import pybamm
+import numpy as np
+import unittest
+
+testcase = []
+if pybamm.have_idaklu() and pybamm.have_jax():
+    from jax.tree_util import tree_flatten
+    import jax
+    import jax.numpy as jnp
+
+    inputs = {
+        "Current function [A]": 0.222,
+        "Separator porosity": 0.3,
+    }
+
+    model = pybamm.lithium_ion.DFN()
```

Comment:
Should use a simpler model here, e.g. exponential decay, so that the test is faster. I don't think using a complicated model has any value since you are re-using the idaklu solver

Reply:
Okay (it was useful for initial testing!), but I've replaced it with a simplified expression set for future CI.